src/app/frontend/replicasetdetail/replicasetdetail.scss

```diff
@@ -53,18 +53,13 @@ $table-cell-height-half: $table-cell-height / 2;
   font-weight: $bold-font-weight;
   padding-bottom: $baseline-grid;
   padding-top: $baseline-grid;
-
-  &.kd-replicasetdetail-sidebar-info {
-    padding-bottom: 0;
-    padding-top: 2 * $baseline-grid;
-  }
 }
 
 .kd-replicasetdetail-sidebar-line {
   color: $foreground-2;
   display: block;
   font-size: $body-font-size-base;
-  padding-bottom: $baseline-grid;
+  padding-bottom: $baseline-grid / 4;
```

Comment:
You know me... I'm opposed to everything that is not a natural multiplier of $baseline-grid :) How about making this 0 and line 70 $baseline grid. This would be pretty much the same as on replica set cards.